deployment/deploy_storybook_gitlab.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+PAGES_BRANCH="gh-pages"
+
+echo "open ssl connection"
```

Comment:
Should probably be something like "setup ssh". No openssl used here and no connection opened.